src/allvars.h

```diff
@@ -1107,9 +1110,11 @@ struct Options
         mpiparticlebufsize=-1;
         mpinprocswritesize=1;
 #ifdef SWIFTINTERFACE
-        impiusemesh = true;
+        impiusemesh = false;
+	impiusetree = true;
 #else
-        impiusemesh = true;
+        impiusemesh = false;
+	impiusetree = true;
```

Comment:
Default configuration should not be changed. Plus swift will use a mpi mesh decomposition so please revert.

Reply:
@JinsuRhee , can update the user interface so that this option can be set at runtime or when a ramses input is loaded?